core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2ServerConfigProvider.java

```diff
@@ -29,14 +29,16 @@ class OAuth2ServerConfig
         private final URI tokenUrl;
```

Comment:
this could be `record`. Maybe follow up?

Reply:
I added it as a prep commit